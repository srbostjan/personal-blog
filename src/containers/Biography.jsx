import React, { useEffect } from 'react';
import HeaderMobile from '../components/Header/HeaderMobile';
import NavDesktop from '../components/Header/NavDesktop';
import DownArrow from '../components/DownArrow';

function Biography() {
  useEffect(() => {
    window.scrollTo(0, 0);
  });
  return (
    <div className="main-biography">
      <HeaderMobile nav="biography" name="Biography" />
      <div className="main-biography__content">
        <div className="biography-header">
          <div />
          <NavDesktop nav="biography" />
        </div>
        <section className="my-info">
          <div className="my-info--presentation">
            <img src="https://i.ibb.co/bRx3FJm/people-guy-man-black-and-white-1.png" alt="" />
            <div className="biography--titles">
              <h3>Sebas Rivera</h3>
              <p>Frontend developer</p>
              <p>UI:UX Enthusiast</p>
            </div>
          </div>
          <section className="my-info__content">
            <section className="my-info--presentation">
              <div className="biography--titles">
                <h3>Sebas Rivera</h3>
                <p className="text">Frontend developer</p>
                <p className="text">UI:UX Enthusiast</p>
              </div>
            </section>
            <div className="resume">
              <h2>Resume</h2>
              <p className="text">Fronted developer y UI:UX enthusiast enfocado en crear c√≥digo limpio y con buenas practicas. especializado en: HTML, CSS, JavaScript, React, Redux, React Native, Vue, Sass, Webpack, Git/Github, UI Design Figma y Cypress.</p>
            </div>
            <div className="stack">
              <h2>Stack</h2>
              <p className="text">Fronted developer y UI:UX enthusiast enfocado en crear codigo limpio y con buenas practicas. especializado en: HTML, CSS, JavaScript, React, Redux, React Native, Vue, Sass, Webpack, Git/Github, UI Design Figma y Cypress.</p>
            </div>
          </section>
        </section>
      </div>
      <DownArrow />
    </div>
  );
}

export default Biography;
