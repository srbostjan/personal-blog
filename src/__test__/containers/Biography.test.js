import React from 'react';
import { mount, shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Biography from '../../containers/Biography';

describe('<Biography />', () => {
  const component = shallow(<Biography />);
  test('Render componente Biography', () => {
    expect(component.length).toEqual(1);
  });
  test('Render Titulo biografia', () => {
    expect(component.find('.biography--titles--title').text()).toEqual('Sebas Rivera');
  });
});
describe('Biography Snapshot', () => {
  test('Biography Snapshot', () => {
    const component = create(<BrowserRouter><Biography /></BrowserRouter>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});