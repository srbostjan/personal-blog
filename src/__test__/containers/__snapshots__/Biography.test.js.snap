// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Biography Snapshot Biography Snapshot 1`] = `
<div
  className="main-biography"
>
  <header
    className="header-mobile"
  >
    <h1
      className="header__title"
    >
      Biography
    </h1>
    <nav>
      <input
        className="nav__checkbox"
        id="nav__checkbox"
        name=""
        type="checkbox"
      />
      <label
        className="nav__toggle"
        htmlFor="nav__checkbox"
      >
        <i
          className="fas fa-bars menu"
        />
        <i
          className="fas fa-times close"
        />
        <input
          className="nav__checkbox"
          id="nav__checkbox"
          name=""
          type="checkbox"
        />
      </label>
      <ul
        className="nav__menu"
      >
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="active"
        >
          <a
            href="/biography"
            onClick={[Function]}
          >
            Biography
          </a>
        </li>
        <li>
          <a
            href="/articles"
            onClick={[Function]}
          >
            Articles
          </a>
        </li>
        <li>
          <a
            href="https://twitter.com/srbostjan"
            rel="noreferrer"
            target="_blank"
          >
            Twitter
          </a>
        </li>
        <li>
          <a
            href="https://github.com/srbostjan"
            rel="noreferrer"
            target="_blank"
          >
            GitHub
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <div
    className="main-biography__content"
  >
    <div
      className="biography-header"
    >
      <div />
      <ul
        className="nav-desktop"
      >
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="active"
        >
          <a
            href="/biography"
            onClick={[Function]}
          >
            Biography
          </a>
        </li>
        <li>
          <a
            href="/articles"
            onClick={[Function]}
          >
            Articles
          </a>
        </li>
        <li>
          <a
            href="https://twitter.com/srbostjan"
            rel="noreferrer"
            target="_blank"
          >
            Twitter
          </a>
        </li>
        <li>
          <a
            href="https://github.com/srbostjan"
            rel="noreferrer"
            target="_blank"
          >
            GitHub
          </a>
        </li>
      </ul>
    </div>
    <section
      className="my-info"
    >
      <div
        className="my-info--presentation"
      >
        <img
          alt="man portrait"
          src="https://i.ibb.co/bRx3FJm/people-guy-man-black-and-white-1.png"
        />
        <div
          className="biography--titles"
        >
          <h3
            className="biography--titles--title"
          >
            Sebas Rivera
          </h3>
          <p>
            Frontend developer
          </p>
          <p>
            UI:UX Enthusiast
          </p>
        </div>
      </div>
      <section
        className="my-info__content"
      >
        <section
          className="my-info--presentation"
        >
          <div
            className="biography--titles"
          >
            <h3>
              Sebas Rivera
            </h3>
            <p
              className="text"
            >
              Frontend developer
            </p>
            <p
              className="text"
            >
              UI:UX Enthusiast
            </p>
          </div>
        </section>
        <div
          className="resume"
        >
          <h2>
            Resume
          </h2>
          <p
            className="text"
          >
            Fronted developer and UI:UX enthusiast focused on creating clean code with good practices. specialized in: HTML, CSS, JavaScript, React, Redux, React Native, Vue, Sass, Webpack, Git/Github, UI Design Figma and Cypress.
          </p>
        </div>
        <div
          className="stack"
        >
          <h2>
            Stack
          </h2>
          <ul
            className="stack-list"
          >
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              React
            </li>
            <li>
              Sass
            </li>
            <li>
              React Native
            </li>
            <li>
              Cypress
            </li>
            <li>
              Git
            </li>
            <li>
              Redux
            </li>
            <li>
              Webpack
            </li>
          </ul>
        </div>
      </section>
    </section>
  </div>
  <button
    className="down-arrow-button"
    onClick={[Function]}
    type="button"
  >
    <i
      className="fas fa-arrow-circle-right down-arrow"
    />
  </button>
</div>
`;
